.github/workflows/release.yml

```diff
@@ -123,27 +133,36 @@ jobs:
         with:
           length: 7
 
+      ##
+      # Uploading artifacts does not retain the file permissions,
+      # binaries should regain the executive permission before packing.
+      # See: https://github.com/actions/upload-artifact#permission-loss
+      ##
       - name: Build and publish docker image
         run: |
           IMAGE_NAME=darwinia
           IMAGE_PREFIX=quay.io/darwinia-network
           IMAGE_PUSH_NAME_TAG=${IMAGE_PREFIX}/${IMAGE_NAME}:${GIT_TAG_NAME}
           IMAGE_PUSH_NAME_SHA=${IMAGE_PREFIX}/${IMAGE_NAME}:sha-${SHA}
 
-          chmod +x shared/darwinia
+          chmod +x shared/darwinia # REQUIRED, see above
           docker build \
-            -t ${IMAGE_NAME} \
+            -t ${IMAGE_PUSH_NAME_TAG} \
+            -t ${IMAGE_PUSH_NAME_SHA} \
+            --build-arg CI_BUILD_AT=${{ needs.build-package.outputs.BUILD_AT }} \
+            --build-arg CI_GIT_TAG=${GIT_TAG_NAME} \
+            --build-arg CI_GIT_SHA=${{ github.sha }} \
             -f .maintain/docker/Dockerfile \
             .
 
-          docker tag ${IMAGE_NAME} ${IMAGE_PUSH_NAME_TAG}
-          docker tag ${IMAGE_NAME} ${IMAGE_PUSH_NAME_SHA}
-
           docker push ${IMAGE_PUSH_NAME_TAG}
           docker push ${IMAGE_PUSH_NAME_SHA}
 
-  complete-build:
-    name: Complete build
+  ##
+  # Publish the release
+  ##
+  publish-release:
```

Comment:
https://github.com/darwinia-network/darwinia/pull/792/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R254
There should be change to `publish-release`